omnibus/cookbooks/firezone/templates/redirect.conf.erb

```diff
@@ -6,6 +6,8 @@ server {
   <% end -%>
   server_name <%= @server_name %>;
 
+  return 301 https://<%= @server_name %>$request_uri;
+
```

Comment:
Verified to work with ACME renewal: 


<img width="1009" alt="Screen Shot 2022-08-30 at 1 42 58 PM" src="https://user-images.githubusercontent.com/167144/187539346-073e712b-e318-4929-9bc2-247fc43410bf.png">

Reply:
I guess the one below is considered more specific